### PR TITLE
manifests: add ipcalc

### PR DESCRIPTION
**Pull request by c4rt0**
See: https://github.com/coreos/fedora-coreos-tracker/issues/1460

### CODE REVIEW

**Review by jlebon — COMMENTED** [review]
You can manually adjust the arches in the non-x86_64 lockfiles.

This is missing the one for ppc64le.

**Comment by dustymabe** [comment]
https://github.com/coreos/coreos-assembler/pull/3478 should partially address the CI flake.

File: manifests/networking-tools.yaml [hunk 1]
```diff
@@ -22,3 +22,5 @@ packages:
   # Declarative network configuration
   # https://github.com/coreos/fedora-coreos-tracker/issues/1175
   - nmstate
+  # Advanced custom networking calculations
```
Comment (dustymabe) [inline manifests/networking-tools.yaml:25]: maybe add a link to the tracker issue discussion here

**Comment by c4rt0** [comment]
> [coreos/coreos-assembler#3478](https://github.com/coreos/coreos-assembler/pull/3478) should partially address the CI flake.

Since it's a work in progress (status "Open") until it's merged I can only watch the tests fail, right?

**Review by dustymabe — APPROVED** [review]

**Comment by dustymabe** [comment]
> > [coreos/coreos-assembler#3478](https://github.com/coreos/coreos-assembler/pull/3478) should partially address the CI flake.
> 
> Since it's a work in progress (status "Open") until it's merged I can only watch the tests fail, right?

It was a flake so it happens sometimes :) - if the tests pass on this run then we can merge this without any issue. I've set it to automerge if the t ests pass.

**Pull request merged**